docs/pages/switch.js

```diff
@@ -141,6 +331,50 @@ function SwitchExample() {
           />
         </MainSection.Subsection>
       </MainSection>
+
+      <MainSection name="Writing">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description={`
+- Be clear and brief with Switch labels so they can be easily understood.
+- When possible, use verb to clarify the action. Something like “set…” or “show…”
```

Comment:
Similar to above, these either all need to end with periods or all not (including this one with the ellipsis)

Reply:
Done!